src/formatDate.js

```diff
@@ -8,7 +8,30 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
```

Comment:
Manipulating the `fromFormat` array with `pop` directly affects the input array, which might not be intended. Consider creating a copy of the array before using `pop` to avoid side effects.